packages/venia-concept/src/components/Checkout/receipt.js

```diff
@@ -12,17 +13,33 @@ class Exit extends Component {
             footer: string,
             root: string
         }),
-        resetCheckout: func.isRequired
+        resetCheckout: func.isRequired,
+        orderId: integer
     };
 
+    static defaultProps = {
+        //TODO: implement fetching of orderId
+        orderId: 777
+    }
+
     render() {
-        const { classes, resetCheckout } = this.props;
+        const { classes, resetCheckout, orderId } = this.props;
 
         return (
             <div className={classes.root}>
-                <div className={classes.body}>Thank you for your order!</div>
-                <div className={classes.footer}>
+                <div className={classes.body}>
+                    <h2 className={classes.headerText}>
+                        Thank you for your purchase!
+                    </h2>
+                    <div className={classes.textBlock}>
+                        Your order # is <span className={classes.orderId}>{orderId}</span><br/>
+                        We\'ll email you an order confirmation with details and tracking info
+                    </div>
                     <ResetButton resetCheckout={resetCheckout} />
```

Comment:
We do not need reset button here. Just implement a CustomButton component for our page, see comment below.